.github/workflows/release_go.yml

```diff
@@ -0,0 +1,64 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+name: Release Go
+
+on:
+  workflow_dispatch:
+    inputs:
+      opendal_core_version:
+        description: "Version of Apache OpenDAL Core"
+        required: true
+        type: string
+      opendal_go_version:
+        description: "Version of Apache OpenDAL Go Binding"
+        required: true
+        type: string
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}-${{ github.event_name }}
+  cancel-in-progress: true
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: ${{ inputs.opendal_core_version }}
+      - name: Tag Go Binding
+        env:
+          TAG: bindings/go/${{ inputs.opendal_go_version }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          git config --global user.name "Github Actions"
+          git config --global user.email "actions@github.com"
+          git tag $TAG
+          git push origin $TAG
+      - name: Release Go Services
+        env:
+          GH_TOKEN: ${{ secrets.GO_SERVICE_TOKEN }}
```

Comment:
@Xuanwo We need a token from https://github.com/apache/opendal-go-services with the permission set of `"Contents" repository permissions (write)`. FYI: https://docs.github.com/en/rest/repos/repos?apiVersion=2022-11-28#create-a-repository-dispatch-event

Can you help set it up for 1Password?

Reply:
Can we use GITHUB_TOKEN here?

Reply:
No. I've tried. the `GITHUB_TOKEN` only has permission that belongs to this repository and others are out of scope.

Reply:
Thanks!